.storybook/preview.css

```
@@ -0,0 +1,20 @@
+@font-face {
+  font-family: "Pretendard";
```

Stylelint (17.11.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)